libs/@blockprotocol/graph/src/internal/mutate-subgraph/element.ts

```diff
@@ -51,29 +43,27 @@ export const addDataTypesToSubgraphByMutation = (
     subgraph.vertices[baseUrl] ??= {};
     subgraph.vertices[baseUrl]![version] = dataTypeVertex;
 
-    /** @todo - with the introduction of non-primitive data types edges will need to be added here */
+    vertexIds.push({ baseId: baseUrl, revisionId: version.toString() });
   }
   /* eslint-enable no-param-reassign */
+  return vertexIds;
 };
 
 /**
  * Looking to build a subgraph? You probably want {@link buildSubgraph} from `@blockprotocol/graph/stdlib`
  *
- * This MUTATES the given {@link Subgraph} by adding a given list of property types to the vertices, creating any ontology
- * related edges that are **directly implied** by them (see note below).
+ * This MUTATES the given {@link Subgraph} by adding a given list of property types to the vertices..
```

Comment:
```suggestion
 * This MUTATES the given {@link Subgraph} by adding a given list of property types to the vertices.
```

Unless that's for a reason 😁